spec/controllers/feedbacks_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe FeedbacksController, type: :controller do
+  describe "#create" do
+    before do
+      allow(controller).to receive(:level_of_help).and_return("controlled")
```

Comment:
Rather than fake this, I think it's worth creating a session and passing an assessment_code param in your tests.

Reply:
to chat about when pairing